src/code.cloudfoundry.org/policy-server/store/migrations/v0075.go

```diff
@@ -0,0 +1,15 @@
+package migrations
+
+var migration_v0075 = map[string][]string{
+	"mysql": []string{
+		`ALTER TABLE security_groups DROP PRIMARY KEY;`,
```

Comment:
Is this necessary to modify the column? When i looked in MySQL docs, it indicated removing the primary key and having no other key would result in an error. I also have the same question on the postgres side.

Reply:
@geofffranks I've checked it on both cases (PostgreSQL & MySQL) on a test database and can confirm, that the drop of the primary keys is not necessary. I've removed the corresponding update commands.